src/index.js

```diff
@@ -0,0 +1,54 @@
+import './style.css';
+import '@fortawesome/fontawesome-free/js/all.js';
+import { taskInput, ulContainer, btndeleteTask } from './modules/selectors';
+import {
+  objTask, listofTasks, newTask, printTask,
+} from './modules/ui';
+
+function saveTask(e) {
+  objTask[e.target.name] = e.target.value;
+  objTask.complete = false;
+}
+
+// Events
+function eventListeners() {
+  taskInput.addEventListener('input', saveTask);
+}
```

Comment:
Referring to [this article](https://medium.com/infancyit/kiss-keep-it-simple-smart-javascript-habits-b019ce14b5ec) about KISS it is recommended to use arrow functions instead of declaring functions using a `function` keyword. So you can change your code to a single line of code like this: 
`const eventListeners = () => taskInput.addEventListener('input', saveTask);`